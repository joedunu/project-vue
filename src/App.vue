<template>
    <div id="app">
        <NavBar/>
        <router-view/>
        <Footer />
    </div>
</template>

<script>
import NavBar from './components/common/navBar/NavBar.vue';
import Footer from './components/common/footer/Footer.vue';

export default {
  name: 'app',
  components: {
    NavBar,
    Footer
  }
};
</script>

<style>
#app {
  font-family: 'Avenir', Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
}
</style>
