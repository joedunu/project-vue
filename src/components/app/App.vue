<template>
    <div id="app">
        <NavBar/>
        <router-view/>
        <PageFooter />
    </div>
</template>

<script>
import NavBar from "../common/navBar/NavBar"
import PageFooter from "../common/pageFooter/PageFooter"

export default {
	name: "app",
	components: {
		NavBar,
		PageFooter
	}
}
</script>

<style>
#app {
	font-family: "Avenir", Helvetica, Arial, sans-serif;
	-webkit-font-smoothing: antialiased;
	-moz-osx-font-smoothing: grayscale;
	text-align: center;
	color: #2c3e50;
}
</style>
