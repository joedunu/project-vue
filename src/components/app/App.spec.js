import { shallow, createLocalVue } from "@vue/test-utils"
import VueRouter from "vue-router"

import App from "./App.vue"

describe("App", () => {
	test("snapshot: App", () => {
        const localVue = createLocalVue()
        localVue.use(VueRouter)

        let cmp = shallow(App, {localVue})
		expect(cmp.element).toMatchSnapshot()
	})
})
