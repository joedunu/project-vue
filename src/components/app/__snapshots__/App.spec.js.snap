// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot: App 1`] = `
<div
  id="app"
>
  <!---->
   
  <!---->
   
  <!---->
</div>
`;
